backend/routes/event.js

```diff
@@ -87,4 +88,24 @@ router.delete("/:id", auth, (req, res, next) => {
 
 // Should we allow updation to events, requires disussion
 
+router.put("/:id", auth, (req, res) => { 
+    const event = Event.findOne({ _id: req.params.id })
+                        .then(() => {
+                            event.isUpdated = true;
+                            event.title = req.body.title;
+                            event.community = req.body.community;
+                            event.description = req.body.description;
+                            event.pdfPath = req.body.pdfPath;
+                            event.eventLink = req.body.eventLink;
+                            event.eventDate = req.body.eventDate;
+                            event.banner = req.body.banner;
```

Comment:
You forgot to return the response!
return this event with a message OK

Reply:
Woops. Completely forgot this. Thanks!